artifacts/ubuntu/ubuntu_test.go

```diff
@@ -15,7 +15,7 @@ import (
 var _ = Describe("Ubuntu", func() {
 	DescribeTable("Inspect should be able to parse checksum files",
 		func(release, mockFile string, details *api.ArtifactDetails, envVariables map[string]string, metadata *api.Metadata) {
-			c := New(release, envVariables)
+			c := New(release, "x86_64", envVariables)
```

Comment:
No tests for aarch64?

Reply:
I don't think it's needed because there is no different functionality executed if you enter `aarch64`. This value was hardcoded before and now it's passed from here.